.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -47,9 +47,9 @@ RUN git checkout "${GLIB_MUS_SHA}"
 RUN chmod a+x compile-x86_64-alpine-linux.sh
 RUN sh compile-x86_64-alpine-linux.sh
 
-FROM docker.io/library/python:3.12.7-alpine3.20@sha256:5049c050bdc68575a10bcb1885baa0689b6c15152d8a56a7e399fb49f783bf98 as final
+FROM docker.io/library/python:3.13.1-alpine3.20@sha256:e75706bf1e6b27311be500015333c8d3e05a9b4ca3908253d490a1889ab632ef as final
```

Comment:
I don't know why the python version is updating, but I think minor version bump is fine.